src/styles/transitions/_text-page-load.scss

```diff
@@ -0,0 +1,22 @@
+.text-page-load {
+  max-width: 590px;
+  display: block;
+  margin: auto;
+}
+
+.text-page__title-load{
+  height: 32px;
+  margin: 0 0 0.5rem 0;
+  border-radius: 10px;
+  background-color: $light-grey;
+  @media (max-width: $small) {
+      margin: 0.75rem 0;
+    }
+}
+
+.text-page__html-load{
+  height: 700px;
```

Comment:
Instead of a big single block, I think it will look better as a number horizontal rectangles with shorter heights. Also, it doesn't have to be so tall. Maybe three or four line-height rectangles. Also, try varying the widths on them to make it resemble left-aligned text.